client/src/components/Buttons.jsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
+import SeeTheFit from './SeeTheFit.jsx';
 
 class Buttons extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      activeButton: 1
+      activeButton: 1,
+      displayModal: false
     }
     this.handleClick = this.handleClick.bind(this);
+    this.showModal = this.showModal.bind(this);
```

Comment:
Hey George couldnt this be handled with just one function to toggle the state of the button?

Reply:
Thanks for the attention to detail. I made them two separate functions because showModal gets attached to the see the fit button, and hideModal gets attached to the x in the top right corner of the modal.

Reply:
Ok that makes sense, I think there’s a case to be made for having them separate as well.